src/utils_request.js

```diff
@@ -49,6 +49,10 @@ const DEFAULT_HTTP_REQUEST_OPTIONS = {
  *  Function accepts `response` object as a single parameter and should return true or false.
  *  If function returns true request gets aborted. This function is passed to the
  *  [@apify/http-request](https://www.npmjs.com/package/@apify/http-request) NPM package.
+ * @property {boolean} [useHttp2=false]
+ * If set to true, Got will additionally accept HTTP2 requests.
```

Comment:
I would not say Got here, the underlying implementation is irrelevant. Or at least it should be.

Reply:
It?

Reply:
yeah, sure